server/src/Ogmios/Query.hs

```diff
@@ -0,0 +1,91 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
+
+module Ogmios.Query (
+  makeRequest,
+  tryQueryUntilZero,
+  RequestOption (Query, Acquire),
+  defaultServerParameters,
+  ServerParameters (..),
+  QueryOption (..),
+) where
+
+import Control.Exception.Base (SomeException, try)
+
+import Data.Bifunctor (first)
+import Data.ByteString.Lazy (ByteString)
+import Data.Text (Text)
+import Network.Socket (withSocketsDo)
+import Network.WebSockets (ClientApp)
+import Network.WebSockets qualified as WebSockets
+import System.IO (hFlush, stdout)
+import System.Time.Extra qualified as Time.Extra
+
+--------------------------------------------------------------------------------
+
+data QueryOption = CurrentProtocolParameters
```

Comment:
I don't think we need a `QueryOption` type, since we won't be implementing any other Ogmios queries

Reply:
I wrote them like that just in case, but now that I have confirmation that those won't be needed I'm deleting them.